pyproject.toml

```diff
@@ -0,0 +1,149 @@
+[build-system]
+requires = ["flit_core >=3.11,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "deepdiff"
+version = "8.4.2"
+dependencies = [
+  "orderly-set>=5.3.0,<6",
+]
+requires-python = ">=3.8"
+authors = [
+  { name = "Seperman", email = "sep@zepworks.com" }
+]
+maintainers = [
+  { name = "Seperman", email = "sep@zepworks.com" }
+]
+description = "Deep Difference and Search of any Python object/data. Recreate objects by adding adding deltas to each other."
+readme = "README.md"
+license = {file = "LICENSE"}
+keywords = []
+classifiers = [
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
+  "Topic :: Software Development",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "Development Status :: 5 - Production/Stable",
+  "License :: OSI Approved :: MIT License"
+]
+
+# `dependency-groups` would make this a lot cleaner, in theory.
+[project.optional-dependencies]
+coverage = [
+  "coverage~=7.6.0"
+]
+cli = [
+  "click~=8.1.0",
+  "pyyaml~=6.0.0"
+]
+dev = [
+  "bump2version~=1.0.0",
+  "jsonpickle~=4.0.0",
+  "ipdb~=0.13.0",
+  "numpy~=2.2.0; python_version >= '3.10'",
+  "numpy~=2.0; python_version < '3.10'",
+  "python-dateutil~=2.9.0",
+  "orjson~=3.10.0",
+  "tomli~=2.2.0",
+  "tomli-w~=1.2.0",
+  "pandas~=2.2.0",
+  "polars~=1.21.0",
+]
+docs = [
+  # We use the html style that is not supported in Sphinx 7 anymore.
+  "Sphinx~=6.2.0",
+  "sphinx-sitemap~=2.6.0",
+  "sphinxemoji~=0.3.0"
+]
+static = [
+  "flake8~=7.1.0",
+  "flake8-pyproject~=1.2.3",
+  "pydantic~=2.10.0",
+  "types-setuptools~=75.8.0; python_version > '3.8'",
+]
+test = [
+  "pytest~=8.3.0",
+  "pytest-benchmark~=5.1.0",
+  "pytest-cov~=6.0.0",
```

Comment:
Please add `"tox>=4.0,<5.0"` here!

Reply:
This actually doesn't make sense since this subpackage is used when installing software as part of CI. The version checks should be in `noxfile.py`, `pyproject.toml`, `tox.ini`, etc.